spec/factories/addresses.rb

```diff
@@ -5,7 +5,7 @@
     address_line_two { Faker::Address.street_address }
     city { Faker::Address.city }
     county { Faker::Address.city }
-    postcode { Faker::Address.zip }
+    postcode { ["SW10 9LB", "W6 0LQ", "SW1A 1AA", "RG2 7PU", "BH22 7HR"].sample }
```

Comment:
might be nice to configure faker for UK and use its locale specific postcode helper - have not tried this but


[Customize faker locale](https://github.com/faker-ruby/faker#customization)
Supposedly we can then [use this postcode helper](https://faker.readthedocs.io/en/master/locales/en_GB.html)

Reply:
Just checked this and our `Faker::Config.locale` is set to `en-GB` and so faker can generate uk postcodes using

```ruby
Faker::Address.postcode
```

Reply:
Noted for next time